packages/documentation/src/pages/getting-started/common-tasks/unit-tests.mdx

```diff
@@ -0,0 +1,137 @@
+## Unit Tests
+
+#### Where they live
+- Tests can live anywhere in the `src` directory, but in general are located in a directory close to the code or application that they are written to test.
+- The `.unit.spec.js` file extension is used for unit tests, while `.e2e.spec.js` is used for end-to-end (E2E) tests.
+- Visual regression test config has its own directory under `src/platform/testing/visual-regression`
+- Test commands are specified in `package.json`, as well as in vets-website's `README.md`
+- Configuration files for nightwatch and sass-lint live under `config/`
+- Scripts for running nightwatch tests live under `script/`
+
+#### When they run/ how they run
+There are 5 times tests can run:
+
+  1. At any time during local development by running one of the npm scripts specified in `package.json`
+    - Any tests can be run during this time
+  2. During the pre-commit hook
+    - Only linting is run automatically
+  3. Before merging a branch to master via Jenkins
+    - Unit tests, e2e tests, and code climate are automatically run during this time
+  4. Before merging a branch to production via Jenkins
+    - Accessibility tests are automatically run during this time
+  5. After merging to production (this is planned for removal)
```

Comment:
Are we still doing this?

Reply:
I think 4 and 5 can be collapsed into one: "After merging to master"